test/system/registration_test.rb

```diff
@@ -4,6 +4,13 @@ class RegistrationTest < ApplicationSystemTestCase
   setup do
     @user = create(:staff)
     @organization = @user.organization
+    @page_text = create(:page_text, organization: @organization)
```

Comment:
For these 3 system tests, we can remove `Current.organization = @organization`. 
Also, to DRY it up a little, given these are required to set up the PageText that is called in the root page, we could move this logic to a helper method, and just call that method in each setup. That will make any future changes to the page text easier to update in the test setup.

You should be able to create a new file in `test/helpers` that is a module, name something like PageTextSetup or the likes, and put a method in there e.g., `def set_up_page_text` and include that module in these system test classes, then call that method in the setup.